Accompany/Controllers/Core/Home/TodoListViewController.swift

```diff
@@ -135,4 +137,14 @@ extension TodoListViewController: UITableViewDataSource {
     tableView.reloadData()
   }
   
+  func prepareLabel(label: UILabel, string: String, superScript: String){
```

Comment:
Since it's a helper function for viewDidLoad(), please make this function "private". And please try to add some spacing for your codes, it will be more readable.
```suggestion
  private func prepareLabel(label: UILabel, string: String, superScript: String) {
```